src/snapred/backend/recipe/algorithm/GroupDiffractionCalibration.py

```diff
@@ -275,24 +276,30 @@ def PyExec(self) -> None:
             InstrumentWorkspace=self.wsTOF,
             CalibrationWorkspace=self.DIFCfinal,
         )
-        self.convertAndFocusAndReturn(self.wsTOF, self.outputWStof, "after")
+        self.convertAndFocusAndReturn(self.wsTOF, self.outputWStof, "after", self.units)
```

Comment:
This should always call it with "dSpacing".  As per Malcolm, the output should just always be in d-spacing.

Reply:
Okay, now I have a better understanding of the requirements of this story. I've been so used to hearing the word "flag" used in reference for propagating backend functionality to the frontend UI for user control but in this case, it just meant a simple extra input argument for the method making this task much simpler than I imagined.  Thank you for the heads up!